src/main/java/org/broadinstitute/hellbender/tools/walkers/bqsr/BaseRecalibrator.java

```diff
@@ -104,6 +108,7 @@
      * reflected those sites skipped by the -XL argument.
      */
     @Argument(fullName = KNOWN_SITES_ARG_FULL_NAME, doc = "One or more databases of known polymorphic sites used to exclude regions around known polymorphisms from analysis.", optional = false)
+    @WorkflowInput
     private List<FeatureInput<Feature>> knownSites;
```

Comment:
`@WorkflowInput` works ok with fields of a Collection type? Is this covered by tests?

Reply:
Yes. The TestWDLTool has several args that are collections, including `File` and other types. And the generated test data that is used to validate the WDL for actual tools always generates two values for any collection arg.